tests/src/test/scala/actionContainers/BasicActionRunnerTests.scala

```diff
@@ -260,6 +283,43 @@ trait BasicActionRunnerTests extends ActionProxyContainerTestUtils {
     })
   }
 
+  it should s"export environment variables before initialization" in {
+    val config = testEnvPartition
+    if (!config.skipTest) {
```

Comment:
How can we determine from test run results if some runtime supports a specific feature or not? May be we throw a `TestPendingException` such that test reports call them out as pending